scripts/ci/tags.yaml

```diff
@@ -516,3 +516,23 @@ peripheral_802154:
     - nrfxlib/nrf_rpc/
     - zephyr/modules/hal_nordic/nrf_802154/
     - zephyr/soc/nordic/
+
+bootloader:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - nrf/drivers/hw_cc3xx/
+    - nrf/lib/fprotect/
+    - nrf/lib/hw_unique_key/
+    - nrf/lib/identity_key/
+    - nrf/lib/modem_antenna/
+    - nrf/lib/nrf_modem_lib/
+    - nrf/samples/bootloader/
+    - nrf/subsys/bootloader/
+    - nrf/subsys/fw_info/
+    - nrf/subsys/nrf_security/
+    - nrf/tests/subsys/bootloader/
+    - nrfxlib/crypto/
+    - nrfxlib/nrf_modem/
+    - zephyr/soc/nordic/
```

Comment:
```suggestion
    - zephyr/soc/nordic/
    - zephyr/samples/sysbuild/with_mcuboot/
    - zephyr/tests/subsys/dfu/
    - zephyr/tests/boot/test_mcuboot/
```

Reply:
what you think about adding mcumgr also here?
```
    - zephyr/subsys/mgmt/mcumgr/
    - zephyr/tests/subsys/mgmt/mcumgr/
    - zephyr/samples/subsys/mgmt/mcumgr/
    - zephyr/include/zephyr/mgmt/mcumgr/
```
to run all bootloader CI if any change in mcumgr will be done?

Reply:
The same as above, those paths are depencies for tests/subsys/bootloader.
Maybe I should make this tag name more specific?
Intention is not to mark all bootloader/dfu etc. tests.

Reply:
For those proposed I will suggest to create different tag mgmt or dfu but this is outside of this PR.